htdocs/comm/propal.php

```diff
@@ -2252,10 +2252,16 @@
 		// Linked object block
 		$somethingshown = $form->showLinkedObjectBlock($object);
 
-		// Show links to link elements
-		$linktoelem = $form->showLinkToObjectBlock($object);
-		if ($linktoelem) print '<br>'.$linktoelem;
```

Comment:
The goal of this function is to show links to link the current object to other object.

The function showLinkToObjectBlock() previously contained only implementation of links to order objects. It contains now the implementation of two other elements ( interventions and supplier orders), and the list is subject to be extended to contain all common objects in the future.

As it makes no sense to link some element types to other ones (ex : customer orders to  expense report) I proposed the solution to use the array $restrictlinksto, and include in it only the elements allowed to be linked to the current object, and I started by providing the possibility to link propales to only interventions based on the value of PROPALE_LINK_TO_INTERVENTION.  The default behaviour would show links to all types defined in showLinkToObjectBlock.

The matter is subject to discussion.